src/i18n/en.js

```diff
@@ -449,42 +449,43 @@ You'll find an Email with steps to follow.`,
     go_foward_pages: 'Go forward 5 pages',
   },
   plan_calculator: {
-    banner_for_monthly_deliveries: `Looking for something you cannot find? We recommend you to try with our Custom Made Plans. <Link>CONTACT US</Link> and we'll help you.`,
-    banner_for_prepaid: `Do you have more than 100,000 Subscribers? We recommend you to see our <Link>HIGH VOLUME PLANS</Link>.`,
-    banner_for_subscribers: `Do you have more than 100,000 Subscribers? We recommend you to see our <Link>HIGH VOLUME PLANS</Link>.`,
+    banner_for_monthly_deliveries: `Do you need more Emails? <Link>Write to us</Link>.`,
+    banner_for_prepaid: `Do you need more Credits? We advise you to hire a <Link>Plan by Emails</Link>.`,
+    banner_for_subscribers: `Do you need more Contacts? We advise you to hire a <Link>Plan by Emails</Link>.`,
     banner_for_unknown: ' ',
-    button_back: 'Back to Plans',
-    button_purchase: 'Purchase',
-    button_purchase_tooltip: 'Modify your actual plan features to can continue',
+    button_back: 'Go back',
+    button_purchase: 'Continue',
+    button_purchase_tooltip: 'This is the Plan you currently have! Change it to continue.',
```

Comment:
```suggestion
    button_purchase_tooltip: 'This is your current Plan! Change it to continue.',
```
How about this @jmonacchi ? sounds more natural to me.

Reply:
You're right! Thanks :)